client/js/lounge.js

```diff
@@ -465,7 +471,34 @@ $(function() {
 		return false;
 	});
 
+	sidebar.on("click", ".add-channel", (e) => {
+		const id = $(e.target).data("id");
+		const joinForm = $(`#join_channel_${id}`);
+		joinForm.toggle();
+		return false;
+	});
```

Comment:
Could you focus on the first input when user clicks on <kbd>+</kbd>?

Reply:
done